src/mug/styles/colors.clj

```diff
@@ -0,0 +1,26 @@
+(ns mug.styles.colors)
+
+(def default-bg "#f9f5d7")
```

Comment:
Use maps for this kind of thing. I intend to tweak coloring a bit, so you can leave this part in PR intact, just a tip